### DISCUSSION TITLE
GraphQL sorting search results

### DISCUSSION THREAD

**Discussion by jkmathew** [community/community#24579] [Apps, API and Webhooks]
<p>I am trying to send search query with following parameters</p>
<pre><code>query {
 search(type: USER, query:"language:swift sort:login-asc", first: 100) {
  edges {
    node {
      ... on User {
        avatarUrl
        login
        name
        url
      }
    }
  }
}
}
</code></pre>
<p>The <code>sort:login-asc</code> part seems like no effect. If I change sort field to <code>sort:followers-desc</code> its working. </p>
<p>V4 API doesn’t support sorting search result by <code>name</code> or <code>login</code>?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>The GraphQL search connection is a wrapper around GitHub’s standard search functionality. Because of this, you’re only going to get the same kinds of functionality you can find searching through the web UI:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/559079baab09ad01f64d3b2ae5e350c9b3f7fc73.png" data-download-href="/uploads/short-url/ccW7AesLM8Q32df6VeRRbVdDgTF.png?dl=1" title="Screen Shot 2019-06-26 at 1.02.09 PM.png"><img src="https://user-images.githubusercontent.com/1038121/181085311-671838b4-3492-4bfe-a153-e65fa9233ea1.png" alt="Screen Shot 2019-06-26 at 1.02.09 PM.png" data-base62-sha1="ccW7AesLM8Q32df6VeRRbVdDgTF" width="690" height="299" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/559079baab09ad01f64d3b2ae5e350c9b3f7fc73_2_690x299.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/559079baab09ad01f64d3b2ae5e350c9b3f7fc73_2_1035x448.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/5/559079baab09ad01f64d3b2ae5e350c9b3f7fc73_2_1380x598.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/559079baab09ad01f64d3b2ae5e350c9b3f7fc73_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-06-26 at 1.02.09 PM.png</span><span class="informations">1854×804 129 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I hope that helps!</p>

**Comment by Nosferican** [comment]
<p>I would be incredibly helpful if one could orderby/sort search connections by fields (e.g., createdAt).</p>

**Comment by simon-fermor** [comment]
<p>Thank you.  I was also trying to find what could be used in a search.</p>

**Comment by Nosferican** [comment]
<p>Could we have that a as a feature request? Being able to sort repostories through the search functionality by when they were created.</p>
